docs/2.5.2.md

```diff
@@ -0,0 +1,40 @@
+# ProxySQL 2.5.2 and ProxySQL admin tools (2023-06-12)
+
+| Release date | June 12, 2023 |
+|---|---|
+| Install instructions | [ProxySQL and proxysql-admin](https://docs.percona.com/proxysql/install-v2.html) |
+| Download this version | [ProxySQL 2](https://www.percona.com/download-proxysql) using the [instructions](https://docs.percona.com/proxysql/where-to-download-proxysql.html)|
```

Comment:
Should we add this to the Useful links section as we have in other projects?

Reply:
Done.